vernac/obligations.ml

```diff
@@ -20,6 +20,12 @@ open Pp
 open CErrors
 open Util
 
+(* EJGA: This should go away, no more need for fix_exn  *)
+module Stm = struct
+    let u = (fun x -> x)
+    let get_fix_exn () = u
+end
+
```

Comment:
I see, my point is that information about exceptions would be always added in `Reach.known_state` itself, not in the lower layers, but yeah this part is tricky wrt futures, I need to understand the flow better.